### DISCUSSION TITLE
hidden class not working

### DISCUSSION THREAD

**Discussion by doniyoraloxanov** [tailwindlabs/tailwindcss#10930] [Help]
I would like the nav bar hidden on sm  screen and show on md screen , but its  not showing anything , hidden all the time       
<div class="hidden md:flex ></div>

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) that replicates the unexpected behavior.

**Reply by doniyoraloxanov** [reply]
I'll check it out now, thanks Wongjn
